content/knowledge_base/consensus/avalanche/concepts.md

```diff
@@ -0,0 +1,35 @@
+# Consensus algorithm concepts behind Avalanche
+
+## Snow Algorithm group of protocols
+
+A group of protocols with a strong probabilistic safety guarantee in the presence or Byzantine nodes.
+
+### Description
+The core concept of operating is execution of repeated sampling the network at random, ultimately leading to correct nodes behavior to obtain a common statement (decision, transacation value, outcome). This mechanism effectively brings the system to an irreversible state - meaning that a large potion of the network accepted a decision or a statement. So any such conflicting to that statement would be accepted with a negligible probability ε or smaller.
```

Comment:
```suggestion
### Description

The core concept of operating is execution of repeated sampling the network at random, ultimately leading to correct nodes behavior to obtain a common statement (decision, transaction value, outcome). This mechanism effectively brings the system to an irreversible state — meaning that a large potion of the network accepted a decision or a statement. So any such conflicting to that statement would be accepted with a negligible probability ε or smaller.
```

See Markdown style guidelines:

> Keep extra newlines between paragraphs and sections to make the source easier to read.

Also when using an `em dash` for punctuation, please make sure to use the `—` character which is specifically made for that purpose

Reply:
This description has been updated https://github.com/optakt/optakt.github.io/pull/13/commits/0a1cb902f3417c3bc20f9197df71d3d65a1b6421#diff-7b8cdfdde84c7352f51f3cc642d28c90f9d1283870319c81956812353289fb32R7-R13
But I fixed rest of the places